### ISSUE TITLE
 web support

### ISSUE THREAD

**Issue by emreesen27** [kineapps/flutter_archive#4]
Hello,
is it possible for this library to receive web support in the future?

**Comment by kinex** [comment]
Yes it is possible in future. If you have ideas how this could be implemented let me know. I could of course use the other plugin `archive` for web support.

**Comment by tsinis** [comment]
> Yes it is possible in future. If you have ideas how this could be implemented let me know. I could of course use the other plugin `archive` for web support.

Sadly, archive package doesn't support web yet too...

**Comment by MalteMagnussen** [comment]
👍
